app/Http/Requests/VoteItemRequest.php

```diff
@@ -21,9 +23,20 @@ public function authorize()
      */
     public function rules()
     {
+        if($this->vote_id){
+            $vote = Vote::find($this->vote_id);
+            if($vote){
+                $existedVoteItems = $vote->voteitems()->get()->toArray();
+                $stringOfVoteItemsNames ='';
+                foreach ($existedVoteItems as $voteItem){
+                    $stringOfVoteItemsNames .= $voteItem['name'] . ',';
+                }
+            }
+
+        };
         return [
             'vote_id' => 'required|exists:votes,id|integer',
-            'name' => 'required',
+            'name' => 'required|not_in:' . $stringOfVoteItemsNames,
```

Comment:
I guess that will be error with $stringOfVoteItemsNames if $this->vote_id equals 0 or vote with id is not presented in db